frontend/packages/core/src/Input/text-field.tsx

```diff
@@ -106,6 +144,23 @@ const IconButton = styled(MuiIconButton)({
   },
 });
 
+interface resultOption {
+  id: string;
+  label: string;
+}
+
+interface ResultProps {
+  option: resultOption;
+}
+
```

Comment:
It looks like `resultOption` isn't used anywhere outside of result props. You could do something like
```suggestion
interface ResultProps {
  option: {
    id: string;
    label: string;
  };
}

```

Reply:
ah ok nice